include/arch/arm/cortex_m/mpu/arm_mpu.h

```diff
@@ -43,7 +43,9 @@ struct arm_mpu {
 /* Enable privileged software access to the default memory map */
 #define ARM_MPU_PRIVDEFENA	(1 << 2)
 
-#define REGION_VALID	(1 << 4)
+/* ARM MPU RABR Register */
+/* Region address mask */
+#define REGION_ADDR_MASK	(0xFFFFFFE0U)
 
```

Comment:
Please bring the REGION_VALID back and remove the U (it might be used in assembly code) and parentheses from the  REGION_ADDR_MASK.

I would rename is as REGION_BASE_ADDR_MASK to make things clearer as well.

In the comment the register is called RBAR (http://infocenter.arm.com/help/index.jsp?topic=/com.arm.doc.dui0553a/Cihigffb.html)